specificatie/verblijfplaats.yaml

```diff
@@ -0,0 +1,463 @@
+openapi: 3.0.3
+info:
+  title: Verblijfplaats definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    Adresregel1:
+      type: string
+      description: |
+        Het eerste deel van een adres is een combinatie van de straat en huisnummer.
+      # example: "1600 Pennsylvania Avenue NW"
+      example: "Laan van de landinrichtingscommissie Duiven-Westervoort 26A-3"
+    Adresregel2:
+      type: string
+      description: |
+        Het tweede deel van een adres is een combinatie van woonplaats eventueel in combinatie met de postcode.
+      # example: "Washington, DC 20500"
+      example: "1234AA Nootdorp"
+    Adresregel3:
+      type: string
+      description: |
+        Het derde deel van een adres is optioneel. Het gaat om een of meer geografische gebieden van het adres in het buitenland.
+      example: "Selangor"
+    AdresseerbaarObjectIdentificatie:
+      type: string
+      description: |
+        De verblijfplaats van de persoon kan een ligplaats, een standplaats of een verblijfsobject zijn.
+      example: "0226010000038820"
+    GemeenteVanInschrijving:
```

Comment:
gemeenteVanInschrijving is een waardetabel (code+omschrijving). Deze kan ook al door RvIG geleverd worden

Reply:
aangepast